GermanFilter/sections/specific.txt

```diff
@@ -79,6 +79,8 @@ rauchmeldungen.de,simfans.de,wn.de,cosmopolitan.de,wallstreet-online.de,tierchen
 ! ##.id-Container--proBEEP
 hallo-muenchen.de,az-online.de,lokalo24.de,rosenheim24.de,kreiszeitung.de,nordbuzz.de,mannheim24.de,merkur.de,oktoberfest-live.de,tz.de##.id-Container--proBEEP
 !
+flashscore.de##.otPlaceholder
+flashscore.de##.boxOverContent__bannerLink
```

Comment:
Unfortunately it doesn't work on my end.
Maybe something like this will be fine (works on my end):
`flashscore.de##.boxOverContent__banner`